src/audioplayerfactory.cc

```diff
@@ -65,9 +65,9 @@ void AudioPlayerFactory::reset()
 #endif
   }
 
-  QScopedPointer< ExternalAudioPlayer > externalPlayer( new ExternalAudioPlayer );
+  std::unique_ptr< ExternalAudioPlayer > externalPlayer( new ExternalAudioPlayer );
   setAudioPlaybackProgram( *externalPlayer );
-  playerPtr.reset( externalPlayer.take() );
```

Comment:
They realize that .take() makes the existence of this pointer type pointless, so it is removed.

https://github.com/qt/qtbase/commit/612a01be6513894ab1ec5a36b699a2142ba7f35c